contributed_definitions/nyaml/NXmpes_xps_liquid.yaml

```diff
@@ -0,0 +1,314 @@
+#Pincelli, Rettig, Arora at fhi-berlin.mpg.de, Dobener at hu-berlin.de, 06/2022
+#Draft version of a NeXus application definition for photoemission,
+#It is designed to be extended by other application definitions
+#with higher granularity in the data description.
+
+doc: This is the application definition for multidimensional photoelectron spectroscopy on liquids
+category: application
+NXmpes:
+  ENTRY(NXentry):
+    \@entry(NX_CHAR):
+      doc: "NeXus convention is to use “entry1”, “entry2”, for analysis software to locate each entry."
+    \@default(NX_CHAR):
+      exists: optional
+      doc: "Default path for data display."
+    title(NX_CHAR):
+    entry_identifier(NX_CHAR):
+      exists: recommended
+    start_time(NX_DATE_TIME):
+      doc: "ISO8601 formatted Datetime of the start of the measurement."
+    end_time(NX_DATE_TIME):
+      exists: recommended
+      doc: "ISO8601 formatted Datetime of the end of the measurement."
+    definition:
+      \@version(NX_CHAR):
+      enumeration: ["NXmpes_liquid"]
+    INSTRUMENT(NXinstrument):
+      energy_resolution(NX_FLOAT):
+      SOURCE(NXsource):
+        doc: "The source used to generate the primary photons. Properties refer strictly to parameters of the source, not of the output beam. For example, the energy of the source is not the optical power of the beam, but the energy of the electron beam in a synchrotron and so on."
+        type(NX_CHAR):
+          enumeration:
+            [
+              "Synchrotron X-ray Source",
+              "Rotating Anode X-ray",
+              "Fixed Tube X-ray",
+              "UV Laser",
+              "Free-Electron Laser",
+              "Optical Laser",
+              "UV Plasma Source",
+              "Metal Jet X-ray",
+              "HHG laser",
+            ]
+        name(NX_CHAR):
+        probe(NX_CHAR):
+          doc: "Type of probe. In photoemission it's always photons, so the full NIAC list is restricted."
+          enumeration: ["x-ray", "ultraviolet", "visible light"]
+      BEAM_PROBE(NXbeam):
```

Comment:
This is a group with the name `BEAM_PROBE`, i.e., there may be fields written `BEAM_PROBE[any_name]`. Is this the intention or should it rather be a field named `beam_probe`? In this case it has to be written in lower case.